pwa-devdocs/src/venia-pwa-concept/setup/index.md

```diff
@@ -46,6 +46,8 @@ In the PWA Studio project's root directory, run the following command to install
 yarn install
 ```
 
+**Note:** Please be aware that the project uses `yarn workspaces` and does not support `npm install`. Please use `yarn install` instead.
```

Comment:
@jcalcaben This is the correct place to put this, right?

@revanth0212 We may want to add a note to this repo's `README` as well.

Reply:
Good callout on the README